minecraft_atomizer.py

```diff
@@ -206,6 +212,13 @@ def write_to_csv(filename):
     setup_args()
     imported_csv_items = read_items_from_csv('items.csv')
     atmoize(imported_csv_items, ITEM_NAME)
-    print(raw_materials)
+    if REDUCED:    
+        print("\n───────────────────────────────────")
+        for i in raw_materials:
+            if (raw_materials[i] > 0):
+                print('{:1s} {:18s} {:3s} {:3.5f} {:3s} '.format("│",i," │ ",raw_materials[i]," │ "))
```

Comment:
Please add a one-line comment (I know, heresy!) explaining what this formatting does.